app/src/main/java/com/amaze/filemanager/filesystem/operations/exceptions/CancellationIOException.java

```diff
@@ -0,0 +1,7 @@
+package com.amaze.filemanager.filesystem.operations.exceptions;
+
+import java.io.IOException;
+
+public class CancellationIOException extends IOException {
+
+}
```

Comment:
Should this have a default constructor or an explicitly defined constructor like `ShellNotRunningIOException`?

Reply:
Should this have a default constructor, it does, the default constructor is implicit :P

Reply:
Right, I see a constructor defined for `ShellNotRunningIOException` and `ZipIOException` but not this one. Is there a good reason why?

Reply:
> Right, I see a constructor defined for `ShellNotRunningIOException` and `ZipIOException` but not this one. Is there a good reason why?

Both of those Exception require an exception to be passed as argument. `ShellNotRunningIOException` needs a `ShellNotRunningException` and `ZipIOException` needs a `ZipException`.